docs/reference/stable-patch-format.rst

```diff
@@ -0,0 +1,337 @@
+Stable Patch Format
+===================
+
```

Comment:
https://docs.ubuntu.com/styleguide/en/#headings

We avoid stacked headings (meaning continuous headings without content in between).
Some intro / context about this document would be helpful to readers.

Reply:
Done. Thanks!